doc/sphinx/Pacemaker_Remote/kvm-tutorial.rst

```diff
@@ -293,7 +92,7 @@ Copy the authentication key from a host:
 
     # mkdir -p --mode=0750 /etc/pacemaker
     # chgrp haclient /etc/pacemaker
-    # scp root@example-host:/etc/pacemaker/authkey /etc/pacemaker
+    # scp root@pcmk-1:/etc/pacemaker/authkey /etc/pacemaker
```

Comment:
The add-guest command should do most of this subsection for you (enabling pacemaker_remote, copying the key, starting pacemaker_remote). It might be necessary to install pcs and run pcs host auth on the VM before running add-guest.